src/main/java/no/nav/familie/inntektsmelding/forespørsel/tjenester/ForespørselBehandlingTjenesteImpl.java

```diff
@@ -99,6 +92,78 @@ class ForespørselBehandlingTjenesteImpl implements ForespørselBehandlingTjenes
         return forespørselTjeneste.finnForespørsel(forespørselUUID);
     }
 
+    @Override
+    public void oppdaterAlleForespørslerISaken(Ytelsetype ytelsetype,
+                                               AktørIdEntitet aktørId,
+                                               Map<LocalDate, List<OrganisasjonsnummerDto>> organisasjonerPerSkjæringstidspunkt,
+                                               SaksnummerDto fagsakSaksnummer) {
+        List<ForespørselEntitet> eksisterendeForespørsler = forespørselTjeneste.finnForespørslerForSak(fagsakSaksnummer);
+
+        // Oppretter forespørsler for alle skjæringstidspunkter som ikke allerede er opprettet
+        organisasjonerPerSkjæringstidspunkt.forEach((skjæringstidspunkt, organisasjoner) -> {
+            organisasjoner.forEach(organisasjon -> {
+                Optional<ForespørselEntitet> åpenForespørsel = eksisterendeForespørsler.stream()
+                    .filter(eksisterendeForespørsel -> eksisterendeForespørsel.getSkjæringstidspunkt().equals(skjæringstidspunkt))
+                    .filter(eksisterendeForespørsel -> eksisterendeForespørsel.getOrganisasjonsnummer().equals(organisasjon.orgnr()))
+                    .findFirst();
+
+                if (åpenForespørsel.isEmpty()) {
+                    opprettForespørselOppgave(ytelsetype, aktørId, fagsakSaksnummer, organisasjon, skjæringstidspunkt);
+                    var msg = String.format("Oppretter forespørsel, orgnr: %s, stp: %s, saksnr: %s, ytelse: %s",
+                        organisasjon.orgnr(), skjæringstidspunkt, fagsakSaksnummer.saksnr(), ytelsetype);
+                    LOG.info(msg);
+                }
+            });
+        });
+
+        // Forespørsler som ikke lenger er aktuelle settes til utgått
+        eksisterendeForespørsler.forEach(eksisterendeForespørsel -> {
+            boolean trengerEksisterendeForespørsel = innholderRequestEksisterendeForespørsel(organisasjonerPerSkjæringstidspunkt,
+                eksisterendeForespørsel);
+
+            if (!trengerEksisterendeForespørsel && eksisterendeForespørsel.getSakStatus() == SakStatus.UNDER_BEHANDLING) {
+                //TODO: Trenger vi en ny status i arbeidsgiver-notifikasjon for utgått?
+                arbeidsgiverNotifikasjon.lukkOppgave(eksisterendeForespørsel.getOppgaveId(), OffsetDateTime.now());
+                arbeidsgiverNotifikasjon.ferdigstillSak(eksisterendeForespørsel.getSakId()); // Oppdaterer status i arbeidsgiver-notifikasjon
+                forespørselTjeneste.settSakTilUtgått(eksisterendeForespørsel.getSakId());
```

Comment:
Har ikke lagd noen egen type for `arbeidsgiverNotifikasjon`. Har gjenbrukt `lukkOppgave` og `ferdigstillSak` slik vi bruker de i `ferdigstillForespørsel` ([linje 86](https://github.com/navikt/ft-inntektsmelding/pull/135/files#diff-78f8d57e8c2d197fad39d2a454e827b40d5c2d1652e12662dfd9f708cf520051R86)). Usikker på om vi trenger det, så tar gjerne imot innspill